src/pages/home/report/ReportActionsView.js

```diff
@@ -370,7 +517,7 @@ export default compose(
     withLocalize,
     withNetwork(),
     withOnyx({
-        session: {
+        sesion: {
```

Comment:
This change doesn't look right

```suggestion
        session: {
```

Reply:
hmm seems to be some mess up while resolving conflicts